clguetzli/clguetzli.cl.h

```diff
@@ -0,0 +1,162 @@
+/*
+* OpenCL/CUDA edition implementation of ButteraugliComparator.
+*
+* Author: strongtu@tencent.com
+*         ianhuang@tencent.com
+*         chriskzhou@tencent.com
+*/
+#ifndef __CLGUETZLI_CL_H__
+#define __CLGUETZLI_CL_H__
+
+#ifdef __USE_OPENCL__
+
+#ifdef __cplusplus
+#ifndef __CUDACC__
+#include "CL/cl.h"
+#include "cuda.h"
```

Comment:
I'm having trouble building OpenCL only with the Intel SDK, this seems to be part of it.